commit d4691b4a23ae319b39ae3a84f8435d4dc991bfac
Author: Richard Braun <rbraun@sceen.net>
Date:   Tue Jan 22 00:04:23 2013 +0100

    Fix versions of _IO_flockfile, _IO_ftrylockfile and _IO_funlockfile
    
    * Versions: Include _IO_flockfile, _IO_ftrylockfile and _IO_funlockfile
    in version 2.2.6 of libpthread.

---
 Versions |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Versions b/Versions
index ca975e0..8573637 100644
--- a/libpthread/Versions
+++ b/libpthread/Versions
@@ -22,11 +22,12 @@ libc {
 }
 
 libpthread {
+  GLIBC_2.2.6 {
+    _IO_flockfile; _IO_ftrylockfile; _IO_funlockfile;
+  }
   GLIBC_2.12 {
     __pthread_errorcheck_mutexattr; __pthread_recursive_mutexattr;
 
-    _IO_flockfile; _IO_ftrylockfile; _IO_funlockfile;
-
     __pthread_get_cleanup_stack;
 
     __pthread_mutex_transfer_np;
